scalability_and_performance/using-node-tuning-operator.adoc

```diff
@@ -30,4 +30,4 @@ include::modules/advanced-node-tuning-hosted-cluster.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-For more information about hosted control planes, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)].
+For more information about hosted control planes, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/hosted_intro#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)].
```

Comment:
As mentioned in previous comments, I didn't see this section title in the product doc